option optimize_for = LITE_RUNTIME;

message ItemComposeRequest
{
 required uint32 id = 1;
}
